torch/_tensor_docs.py

```diff
@@ -1840,10 +1840,9 @@ def add_docstr_all(method, docstr):
 
 add_docstr_all('index_put',
                r"""
-index_put(tensor1, indices, values, accumulate=False) -> Tensor
```

Comment:
Is this PR also trying to fix a different doc issue?

Maybe you just need to rebase on master when your other doc fix lands: https://github.com/pytorch/pytorch/pull/60486/files